### PR TITLE
SUNS Part Two: Weaponry

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
Wait a minute! You're not the person that did part one!

I know, she asked me to implment the swords. I did tell her to wait adding the guns because of the hunter's pride PR, but since thats merged, this shouldn't be too big of a deal. The sword sprites were made by Cre.

While factional paint on a syndicate faction's guns is unusual, i felt it was the best way to unifiy the weird mix of Solarian weapons and scarborghs tactical guns. However, I didn't spend much time on the scarborgh weapons as they are due for a resprite anyways.

## Why It's Good For The Game

image

## Changelog

:cl: rye-rice, cre
add: SUNS's swords! Now you can duel over petty matters and potentially die, just like your forefathers!
add: SUNS's guns! Now you can fight over serious matters and potentially die, just like your syndicate comrades!
/:cl:

### CODE REVIEW

**Review by meemofcourse — APPROVED** [review]
twink weapon

**Comment by rye-rice** [comment]
Correct
Small guys, bigass guns with a extreme rate of fire

File: code/modules/projectiles/guns/ballistic/gauss.dm [hunk 1]
```diff
@@ -65,6 +70,11 @@
 	empty_indicator = FALSE
 	manufacturer = MANUFACTURER_SOLARARMORIES
 
+/obj/item/gun/ballistic/automatic/powered/gauss/claris/suns
+	desc = "An antiquated Solarian rifle. Chambered in ferromagnetic pellets, just as the founding Solarians intended, and apparently the SUNS founders as well as it appears painted in the colors of SUNS."
```
Comment (JimKil3) [inline code/modules/projectiles/guns/ballistic/gauss.dm]:
```suggestion
	desc = "An antiquated Solarian rifle. Chambered in ferromagnetic pellets, just as the founding Solarians intended. Evidently, SUNS' founders echo the sentiment, as it appears to be painted in their colors."
```

File: code/modules/projectiles/guns/ballistic/gauss.dm [hunk 2]
```diff
@@ -86,3 +96,8 @@
 /obj/item/gun/ballistic/automatic/powered/gauss/gar/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/automatic_fire, 0.2 SECONDS)
+
+/obj/item/gun/ballistic/automatic/powered/gauss/gar/suns
+	desc = "A Solarian carbine, unusually modern for its producers. However, it is just modern enough for SUNS which also painted the gun in their colors. Launches ferromagnetic lances at alarming speeds."
```
Comment (JimKil3) [inline code/modules/projectiles/guns/ballistic/gauss.dm]:
```suggestion
	desc = "A Solarian carbine, unusually modern for its producers. It's just modern enough for SUNS, however, who have painted the weapon in their colors. Launches ferromagnetic lances at alarming speeds."
```

File: code/modules/projectiles/guns/ballistic/gauss.dm [hunk 3]
```diff
@@ -46,6 +46,11 @@
 	spread = 3
 	spread_unwielded = 12
 
+/obj/item/gun/ballistic/automatic/powered/gauss/modelh/suns
+	desc = "Standard-issue pistol exported from the Solarian Confederation. Fires slow ferromagnetic slugs at a high energy cost, though they rend flesh with ease. It is painted in the colors of SUNS."
```
Comment (JimKil3) [inline code/modules/projectiles/guns/ballistic/gauss.dm]:
```suggestion
	desc = "A standard-issue pistol exported from the Solarian Confederation. It fires slow ferromagnetic slugs at a high energy cost, though they rend flesh with ease. It is painted in the colors of SUNS."
```

File: code/modules/projectiles/guns/ballistic/shotgun.dm [hunk 4]
```diff
@@ -238,7 +237,13 @@
 	pin = /obj/item/firing_pin
 	manufacturer = MANUFACTURER_INTEQ
 
-/obj/item/gun/ballistic/shotgun/bulldog/minutemen
+/obj/item/gun/ballistic/shotgun/bulldog/suns
+	name = "\improper Bulldog-C Shotgun"
+	desc = "A shorter variation of the Bulldog manufactured by Scarborough Arms. A shortened variant for SUNS who require more faster wielding Bulldogs personal defence."
```
Comment (JimKil3) [inline code/modules/projectiles/guns/ballistic/shotgun.dm]:
```suggestion
	desc = "A variation of the Bulldog manufactured by Scarborough Arms for SUNS. Its shorter barrel is intended to provide additional maneuverability in personal defense scenarios."
```
This is a pretty big change from the original wording, but I'm not sure how to make the original sentence structure work.

File: code/modules/projectiles/guns/ballistic/smg.dm [hunk 5]
```diff
@@ -63,6 +63,11 @@
 	icon_state = "cobra20"
 	item_state = "cobra20"
 
+/obj/item/gun/ballistic/automatic/smg/c20r/suns
+	desc = "A bullpup .45 SMG, designated 'C-20r'. Has a 'Scarborough Arms - Per falcis, per pravitas' buttstamp. It is painted in the colors of SUNS."
```
Comment (JimKil3) [inline code/modules/projectiles/guns/ballistic/smg.dm]:
```suggestion
	desc = "A bullpup .45 SMG designated 'C-20r.' Its buttstamp reads 'Scarborough Arms - Per falcis, per pravitas.' This one is painted in SUNS' colors."
```

File: code/game/objects/items/storage/belt.dm [hunk 6]
```diff
@@ -842,6 +842,62 @@
...
+		))
+
+/obj/item/storage/belt/sabre/suns/PopulateContents()
+	new /obj/item/melee/sabre/suns(src)
+	update_appearance()
+
+
+/obj/item/storage/belt/sabre/suns/captain
+	name = "SUNS captain's sabre sheath"
+	desc = "An elegant and impressively made leather sheath designed to hold an captain's blade."
```
Comment (JimKil3) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
	desc = "An elegant and impressively made leather sheath designed to hold a captain's blade."
```

File: code/game/objects/items/storage/belt.dm [hunk 7]
```diff
@@ -842,6 +842,62 @@
 	new /obj/item/melee/sabre/solgov(src)
 	update_appearance()
 
+/obj/item/storage/belt/sabre/suns
+	name = "SUNS sabre sheath"
+	desc = "A leather sheath designed to hold an blade."
```
Comment (JimKil3) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
	desc = "A leather sheath designed to hold a blade."
```
In my opinion, this desc is a little plain - maybe you could do something like...
```suggestion
	desc = "A leather sheath for sabers, commonly worn by SUNS academic staff."
```

File: code/game/objects/items/melee/misc.dm [hunk 8]
```diff
@@ -85,6 +85,18 @@
 	icon_state = "sabresolgov"
 	item_state = "sabresolgov"
 
+/obj/item/melee/sabre/suns
+	name = "SUNS sabre"
+	desc = "A blade of solarian origin given to SUNS followers."
```
Comment (JimKil3) [inline code/game/objects/items/melee/misc.dm]:
```suggestion
	desc = "A blade of Solarian origin given to SUNS followers."
```

File: code/game/objects/items/melee/misc.dm [hunk 9]
```diff
@@ -85,6 +85,18 @@
...
+	icon_state = "suns-sabre"
+	item_state = "suns-sabre"
+
+/obj/item/melee/sabre/suns/captain
+	name = "SUNS captain sabre"
+	desc = "An elegant blade awarded to SUNS captains and officers. Despite the higher craftmanship, it appears to be just as effective as a normal sabre."
```
Comment (JimKil3) [inline code/game/objects/items/melee/misc.dm]:
```suggestion
	desc = "An elegant blade awarded to SUNS captains and officers. Despite its higher craftmanship, it appears to be just as effective as a normal sabre."
```
Looking at the outfits, this blade is only awarded to captains, not officers. Maybe change the desc to reflect that, like...
```suggestion
	desc = "An elegant blade awarded to SUNS captains. Despite its higher craftmanship, it appears to be just as effective as a normal sabre."
```

**Review by JimKil3 — COMMENTED** [review]
Here's a few grammar things I spotted. Other than that, the purple/white color scheme on the guns is definitely unique - I'm looking forward to seeing these in game!

**Comment by rye-rice** [comment]
added in some of the swords i weren't sure what they were until drag clarified, ready again

**Pull request merged**